visidata/loaders/postgres.py

```diff
@@ -68,10 +68,10 @@ def query_async(self, qstr, callback=None):
             cur.close()
 
 
-def cursorToColumns(cur):
+def cursorToColumns(cur, sheet):
     cols = []
     for i, coldesc in enumerate(cur.description):
-        c = ColumnItem(coldesc.name, i, type=codeToType(coldesc.type_code, coldesc.name))
+        c = ColumnItem(coldesc.name, i, type=codeToType(coldesc.type_code, coldesc.name), sheet=sheet)
```

Comment:
Ah, yes, this may fix the immediate issue but leaves the door open for other bugs.  The proper fix is to always use `addColumn` to add a column to a sheet.  That, or calling `sheet.recalc()` after all columns have been added manually.  But I prefer the addColumn method; recalc may be deprecated at some point.